Include/pystate.h

```diff
@@ -51,6 +51,15 @@ typedef struct _is {
 } PyInterpreterState;
 #endif
 
+typedef struct _co_extra_state {
+    struct _co_extra_state *next;
+    PyInterpreterState* interp;
+
+    Py_ssize_t co_extra_user_count;
+    freefunc co_extra_freefuncs[MAX_CO_EXTRA_USERS];
+} PyCodeExtraState;
+
+PyCodeExtraState* _PyCodeExtraState_Get();
```

Comment:
Maybe two underscores / comment that this method is a private API that will go way in 3.7.